Rover/Parameters.h

```diff
@@ -410,6 +410,9 @@ class ParametersG2 {
     // FS options
     AP_Int32 fs_options;
 
+    // Rover options
+    AP_Int32 rover_options;
```

Comment:
I think we should move this variable down below GUID_OPTIONS to keep the order consistent with the Paraemters.cpp file.